setup.cfg

```
@@ -67,7 +67,7 @@ google =
 http =
     apache-airflow-providers-http>=4.9.0
 microsoft.azure =
-    apache-airflow-providers-microsoft-azure>=8.5.1
+    apache-airflow-providers-microsoft-azure>=9.0.0rc1
```

```suggestion
    apache-airflow-providers-microsoft-azure>=9.0.0
```
Apply once RC is released